docs/integrations/data-integrations/scylladb.mdx

```diff
@@ -35,14 +35,19 @@ WITH
   PARAMETERS = {
     "user": "user@mindsdb.com",
     "password": "pass",
-    "secure_connect_bundle": "/home/zoran/Downloads/secure-connect.zip"
+    "host": "127.0.0.1",
+    "port": "9042",
+    "keyspace": "test_data"
   };
 ```
+<Tip>
+The protocol version is set to 4 by default. Should you wish to modify it, simply include "protocol_version": 5 within the PARAMETERS dictionary in the query above.
+</Tip>
+
+## Querying the Keyspace:
```

Comment:
I think this title `## Querying the Keyspace:` is not necessary here. We can remove it to keep the format matching other doc pages.